pkg/cmd/clustertask/delete_test.go

```diff
@@ -39,20 +42,72 @@ func TestClusterTaskDelete(t *testing.T) {
 		dynamicClient  dynamic.Interface
 	}
 
+	clusterTaskData := []*v1alpha1.ClusterTask{
```

Comment:
Should we add tests for v1beta1 as well?

Reply:
I Followed rest of the test files, it was done exactly the same way. 
Ref: https://github.com/tektoncd/cli/blob/master/pkg/cmd/task/delete_test.go#L293-L310

I will change it to `v1beta1`.

Reply:
Lets keep it as it is and change it in other PR.

Reply:
Okay :+1: